zerver/openapi/zulip.yaml

```diff
@@ -15055,6 +15055,15 @@ paths:
                                   type: string
                                   description: |
                                     The topic under which the messages were sent.
+
+                                    For clients that don't support the `empty_topic_name` [client capability][client-capabilities],
+                                    the `""` value is replaced with `"general chat"`.
+
+                                    **Changes**: Before 10.0 (feature level 331), `empty_topic_name`
+                                    client capability didn't exist and empty string as the topic name for
+                                    channel messages wasn't allowed.
```

Comment:
For the last two paragraphs, maybe something like this:

> Note that the `""` topic may have been rewritten by the server to "general chat" depending on the value of the `"empty_topic_name"` client capability.

> **Changes**: The `empty_topic_name` client capability is new in Zulip 10.0 (feature level 331).

Reply:
Done